### PR TITLE
Drag skill

### PR DESCRIPTION
**Pull request by wmcclinton**
This PR adds a Drag skill that moves the platform near another object. The drag sampler returns an (x, y) target location of the platform.

### CODE REVIEW

File: predicators/spot_utils/spot_utils.py [hunk 1]
```diff
@@ -1257,15 +1288,54 @@ def navigate_to_position(self, params: Array) -> None:
...
+                get_localization_state()
 
             # (2) Just move
             self.relative_move(params[0], params[1], params[2])
 
         except Exception as e:
             logging.info(e)
 
+    def drag_arm_control(self, params: Array) -> None:
+        """Simple drag impedence controller."""
```
Comment (nkumar-bdai) [inline predicators/spot_utils/spot_utils.py:1301]: this isn't an impedance controller anymore right? let's update this comment accordingly

**Review by nkumar-bdai — COMMENTED** [review]
Looks pretty good to me overall, modulo a couple minor changes! Great work. Will wait for it to be working before giving final approval

File: predicators/ground_truth_models/spot/nsrts.py [hunk 2]
```diff
@@ -27,8 +27,8 @@ def _move_sampler(spot_interface: _SpotInterface, state: State,
         return np.array([0.5, 0.0, 0.0])
     dyaw = 0.0
     # For MoveToObjOnFloor
```
[the remark this answers is not in the document]
Comment (wmcclinton) [reply predicators/ground_truth_models/spot/nsrts.py:30]: added

**Review by nkumar-bdai — APPROVED** [review]
glad to see this working!

**Pull request merged**